apps/admin-gui/src/app/shared/components/hosts-list/hosts-list.component.html

```diff
@@ -0,0 +1,59 @@
+<div class="card mt-3" [class.hide-table]="exporting" [hidden]="dataSource.filteredData.length === 0 || hosts.length === 0">
+  <div class="card-body">
+    <perun-web-apps-table-options (exportEnd)="exporting = false" (exportStart)="exporting = true" [exporter]="exporter" class="ml-auto"></perun-web-apps-table-options>
+    <table mat-table matTableExporter [dataSource]="dataSource" #exporter="matTableExporter" matSort matSortActive="id" matSortDirection="asc" matSortDisableClear
```

Comment:
This row is too long! I know that you probably see it a lot in whole project. Previously we weren't aware of this problem, but now with every new component we create, we are trying for better code readability. Take every item into new row like it is done for example "members-candidates-list.component.html"